markdown/epilogue.markdown

```diff
@@ -1,39 +1,151 @@
 # Epilogue
 
+> Quality means doing it right when no one is looking.
+>
+> — Henry Ford
+
+Our main goal with this book was promoting elegant uses of the NumPy and SciPy
+libraries, and, we hope, inspiring in you the feeling that quality code is
+something worth striving for, while teaching you how to do effective scientific
+analysis with SciPy.
+
 ## Where to next?
```

Comment:
On top of that style necessity, I do think it needs something between the first paragraph and launching into mailing lists.